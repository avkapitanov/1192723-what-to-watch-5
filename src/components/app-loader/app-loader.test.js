import React from "react";
import renderer from "react-test-renderer";
import AppLoader from "./app-loader";

it(`AppLoader component render correctly`, () => {
  const tree = renderer.create(
      <AppLoader />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
