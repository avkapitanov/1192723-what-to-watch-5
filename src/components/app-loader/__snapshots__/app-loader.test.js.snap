// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoader component render correctly 1`] = `
<div
  style={
    Object {
      "backgroundImage": "linear-gradient(-180deg,#180202 0,#000 100%)",
      "height": "100%",
      "position": "fixed",
      "width": "100%",
    }
  }
>
  <svg
    height="44"
    stroke="#fff"
    style={
      Object {
        "left": "50%",
        "marginLeft": "-22px",
        "marginTop": "-22px",
        "position": "absolute",
        "top": "50%",
      }
    }
    viewBox="0 0 44 44"
    width="44"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fillRule="evenodd"
      strokeWidth="2"
    >
      <circle
        cx="22"
        cy="22"
        r="1"
      >
        <animate
          attributeName="r"
          begin="0s"
          calcMode="spline"
          dur="1.8s"
          keySplines="0.165, 0.84, 0.44, 1"
          keyTimes="0; 1"
          repeatCount="indefinite"
          values="1; 20"
        />
        <animate
          attributeName="stroke-opacity"
          begin="0s"
          calcMode="spline"
          dur="1.8s"
          keySplines="0.3, 0.61, 0.355, 1"
          keyTimes="0; 1"
          repeatCount="indefinite"
          values="1; 0"
        />
      </circle>
      <circle
        cx="22"
        cy="22"
        r="1"
      >
        <animate
          attributeName="r"
          begin="-0.9s"
          calcMode="spline"
          dur="1.8s"
          keySplines="0.165, 0.84, 0.44, 1"
          keyTimes="0; 1"
          repeatCount="indefinite"
          values="1; 20"
        />
        <animate
          attributeName="stroke-opacity"
          begin="-0.9s"
          calcMode="spline"
          dur="1.8s"
          keySplines="0.3, 0.61, 0.355, 1"
          keyTimes="0; 1"
          repeatCount="indefinite"
          values="1; 0"
        />
      </circle>
    </g>
  </svg>
</div>
`;
